pocket_lab/pl_gui_views.c

```diff
@@ -1072,11 +1078,11 @@ int32_t pl_gui_create_attributes_view(lv_obj_t *parent,
 		return ret;
 	}
 
-	/* Create channel select menu */
+	/* Create channel select dropdown and insert channel names */
 	pl_gui_dd_chan_select = lv_dropdown_create(parent);
 	lv_obj_align_to(pl_gui_dd_chan_select, pl_gui_dd_device_select,
 			LV_ALIGN_OUT_RIGHT_MID,
-			10,
+			20,
```

Comment:
Should we create macros for the number -20 here?
Seems like the lv_obj_align_to() function call is present more than once.
In case any changes are required in the future, we only need to change the value of the macro

Reply:
there would be so many macros required for this and would not be ideal practice to do. Adding macros might rathe complicate the things. It is better actually to make changes directly into each view and adjust sizes if required in the future. Ideally once GUI design is fixed, there should be minimal or no changes w.r.t widgets sizes.

Reply:
That makes sense.